src/components/List.js

```diff
@@ -82,11 +85,15 @@ You cannot undo this action, and this item's purchase history will be lost.`,
                     className="checkbox"
                   />
                   {result.name}
-                  <button onClick={() => handleDelete(result)}>x</button>
+                  <button onClick={() => setDetails(result)}>Details</button>
+                  <button onClick={() => handleDelete(result)}>Delete</button>
                 </li>
               );
             })}
         </ul>
+        {details.name ? (
```

Comment:
This absolutely works fine, but I noticed we have `lodash` in the project and it comes with an `isEmpty` method which might be more explicit than using `details.name`. I think it would be `import { isEmpty } from 'lodash` at the top, and then `!isEmpty(details)` in the ternary on line 94. I definitely think I'm being nitpicky because you both did awesome on this issue , but wanted to bring up another solution!

Reply:
Yeah, I like this idea^. Not a deal-breaker, for sure, but it's nice to not just rely on the name.

Another way to do this would be 
`{details.name && (
     <Details details={details} setDetails={setDetails} />
   )}`

...no ternary.

Reply:
I haven't used lodash but was looking at isEmpty earlier this week! I'll give it a look and implement.